test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1 @@
+﻿[assembly: NUnit.Framework.Parallelizable(NUnit.Framework.ParallelScope.Fixtures)]
```

Comment:
You could include a `using NUnit.Framework;` and then this line could be simplified to `[assembly: Parallelizable(ParallelScope.Fixtures)]`.

This is fine for now though.